azerothcore/azerothcore-wotlk
# fix(DB/Instance): Fix equip issue and text errors

## CHANGES PROPOSED:
-  Remove Frostmourne as a base equipped item for The Lich King (ID: 37226), add SmartScript to equip Frostmourne when spell Frostmourne Equip (ID: 72729) is used by the cpp
- Unsheathe Frostmourne when The Lich King (ID: 36954) is fighting Jaina/Sylvanas
- Fix spelling errors and incorrect text so it matches the voice lines before the Escape from Arthas gauntlet event

## ISSUES ADDRESSED:
- When initiating the roleplay for Halls of Reflection, The Lich King walks out with Frostmourne already in-hand, when he should walk in unarmed, and walk out with the sword.
- The Lich King fights Jaina/Sylvanas unarmed
- Spelling errors and text errors from The Lich King, Jaina and Sylvanas.
- [Frostmourne before fix](https://youtu.be/87imZWKtO70)


## TESTS PERFORMED:
- Applied SQL
- Entered Halls of Reflection and initiated roleplay for Falric & Marwyn
- Completed Falric & Marwyn, went to Shadow Throne to watch fixed roleplay text and Frostmourne in The Lich King's hand
- [Frostmourne after fix](https://youtu.be/QjXVfqd10Kw)

## HOW TO TEST THE CHANGES:
Unfortunately, because of the attunements, you will have to attune yourself via completing the Forge of Souls & Pit of Saron first.  

**DO NOT** enter with `.gm on` - you will load the incorrect NPCs and cannot complete the quests or see the fixes

Apply SQL, restart server, create two characters - one horde, one alliance, and `.level 79` to become level 80.

**Alliance:**

1. `.quest add 24510`
2. `.tele FrozenHalls`
3. Enter Forge of Souls, `.mod speed 10`
4. `.learn 27261`, put it on easy hotkey - kill everything, complete the quests
5. Enter Pit of Saron, repeat step 4
6. Enter Halls of Reflection, complete roleplay, watch how Arthas enters the room unarmed and equips Frostmourne when he reaches for it
7. Complete Falric, Marwyn and all trash mobs
8. Go to the Shadow Throne, confirm that Arthas has Frostmourne equipped, and read the updated text - [compare with this](https://wow.gamepedia.com/Escape_from_Arthas#Quotes)

**Horde:**

1. `.quest add 24506`
2. `.tele FrozenHalls`
3. Enter Forge of Souls, `.mod speed 10`
4. `.learn 27261`, put it on easy hotkey - kill everything, complete the quests
5. Enter Pit of Saron, repeat step 4
6. Enter Halls of Reflection, complete roleplay, watch how Arthas enters the room unarmed and equips Frostmourne when he reaches for it (Jaina and Uther have scuffed up scripts here, I'm working on fixing that for another PR)
7. Complete Falric, Marwyn and all trash mobs
8. Go to the Shadow Throne, confirm that Arthas has Frostmourne equipped, and read the updated text - [compare with this](https://wow.gamepedia.com/Escape_from_Arthas#Quotes)

Finally, compare the way Frostmourne is handled [with this video.](https://youtu.be/izx-lkkSKA4?t=187)


## Target branch(es):
